### ISSUE TITLE
Document CLI option `--color` (provided by chalk)

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1479]
Why?
I use `npm scripts` for building projects and use https://www.npmjs.com/package/npm-run-all for parallel runs linting, but if you use flag `-l` colors not work. https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md#known-limitations

**Comment by jeddy3** [comment]
Adding `--color` currently works e.g.

```js
{ 
  "scripts": {
    "lint:js": "eslint --ignore-path .gitignore --fix .",
    "lint:css": "stylelint \"web_modules/**/*.css\" --color",
    "lint": "npm-run-all -l --parallel lint:*",
  }
}
```

Can you clarify what you're proposing?

**Comment by evilebottnawi** [comment]
@jeddy3 ooo good, but we not have it is in `--help`

**Comment by jeddy3** [comment]
What's the normal convention for showing these types of flags in the docs and/or `--help`? Am I right in thinking this flag doesn't actually have anything to do with stylelint... it just gets passed long?

**Comment by evilebottnawi** [comment]
@jeddy3 And in docs and in `--help`, i created issue because i did not know about it, however i am in stylelint core :smile: . Eslint show all flags, also in the future people can create issue again asking it, not everyone will be looking code.

**Comment by jeddy3** [comment]
So, it doesn't get used and it just gets passed along to `chalk`? 

But you're advocating adding to the docs and `--help` anyway so people are made aware that you can pass options along? 

I guess I'm easier either way. @davidtheclark Do you have any thoughts on this, or shall we just add it in?

**Comment by davidtheclark** [comment]
Sure, let's add it to the docs.

**Comment by evilebottnawi** [comment]
Done in https://github.com/stylelint/stylelint/pull/1570
